### ISSUE TITLE
Add links to other JS projects

### ISSUE THREAD

**Issue by cmbankester** [immense/immense.github.io#1]
Since http://immense.github.io is the page that displays for http://immense.js.org, I believe it makes sense to turn it into a directory listing of our other JS projects.

As of now, our JS projects are:
- [immybox](https://github.com/immense/immybox)
- [knockout-datatable](https://github.com/immense/knockout-datatable)
- [knockout-pickadate](https://github.com/immense/knockout-pickadate)
- [knockout-pickatime](https://github.com/immense/knockout-pickatime)
- [knockout-tinymce](https://github.com/immense/knockout-tinymce)
- [knockout-quill](https://github.com/immense/knockout-quill)
- [quill-drag-and-drop-module](https://github.com/immense/quill-drag-and-drop-module)

We also have the excellent [php-macaroons](https://github.com/immense/php-macaroons) PHP project which I would love to see displayed somewhere (perhaps as a link in the footer?).

Of course, this will involve adding gh-pages to each of our other projects.

**Comment by cmckni3** [comment]
Material design lite all of the things?

**Comment by cmbankester** [comment]
Or whatever. I went with MDL Lite for http://immybox.js.org because I wanted to see what all the fuss was about.

**Comment by cmckni3** [comment]
I just think MDL looks nicer for forms.

It really doesn't matter what we use for these pages. Just pick something and be consistent with it.

**Comment by cmbankester** [comment]
Agreed. Seems like our options are:
1. Fully custom
2. [Bootstrap](http://getbootstrap.com/)
3. [Zurb Foundation](http://foundation.zurb.com/) (which actually looks really MDL-y these days)
4. [MDL](https://www.google.com/design/spec/material-design/introduction.html)
   - E.g. [MDL Lite](http://www.getmdl.io/)
5. Something else?

**Comment by cmckni3** [comment]
I like MDL

Bootstrap 4 would be awesome but it still has a lot of work left. https://github.com/twbs/bootstrap/pull/17021

**Comment by cmckni3** [comment]
Is there some way we could just have a single showcase but also utilize http://immense.js.org just for the JS projects?

**Comment by cmbankester** [comment]
Yeah, I'm sure there is. Currently, http://immense.js.org just points to this project. Seemed sensible at the time.
